bundles/org.openhab.binding.bluetooth.ruuvitag/src/main/resources/OH-INF/thing/ruuvitag.xml

```diff
@@ -75,7 +75,7 @@
 	</channel-type>
 	<channel-type id="ruuvitag_movementCounter" advanced="true">
 		<item-type>Number:Dimensionless</item-type>
-		<label>Measurement Counter</label>
+		<label>Movement Counter</label>
```

Comment:
Please also fix the entry in the default translations properties.

Reply:
https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.bluetooth.ruuvitag/src/main/resources/OH-INF/i18n/bluetooth.properties#L20

Reply:
https://github.com/ssalonen/openhab2-addons/commit/93c9c6dd1962e77a78c9b9d4f141b18c34902208